portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,7 +40,16 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I'm not familiar with how servlets work. If an IOException is thrown, does it cause your server to crash? Or just the one request to fail?

Reply:
I'm not 100% sure, but I think it just causes that one request to fail.

Reply:
Can you verify that this is the case?

You can add a line in your method `throw new IOException()` to test this

Reply:
I threw the error in doGet() for the comments servlet and that gave me a 500 internal server error, but the song recommendation form GET and POST work fine.

Reply:
Great! So it seems like it just fails the one request and your server is still running.